Homework 29

xXFranzFXx opened the pull request:
Franz Fernando A51 homework 29

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates addSongToPlayist test case

xXFranzFXx committed:
syntax corrections for chromedriver

xXFranzFXx committed:
homework 18 initial commit

xXFranzFXx committed:
sets up @Test playSong method, configures TestNG xml

xXFranzFXx committed:
sets up helper methods for playSong test case

xXFranzFXx committed:
creates playSong testCase

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
updates homework 19

xXFranzFXx committed:
creates deletPlaylist test case

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
refactors homework 19 with selenium waits

xXFranzFXx committed:
refactors basetest  class

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates renamePlaylist method

xXFranzFXx committed:
Creates BasePage Object

xXFranzFXx committed:
Creates HomePage object

xXFranzFXx committed:
creates login test using POM

xXFranzFXx committed:
Syntax corrections

xXFranzFXx committed:
refactors BasePage to SPF

xXFranzFXx committed:
refactors LoginPage to SPF

xXFranzFXx committed:
Refactors HomePage and LoginPage for SPF

xXFranzFXx committed:
reuses homework22 for SPF test case

xXFranzFXx committed:
initial commit

sets up selenium grid

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
initial commit

sets up lambdatest

xXFranzFXx committed:
homework-25

sets up lambdatest for selenium parallel testing

xXFranzFXx committed:
syntax corrections

removes unnecessary files, makes syntax corrections to BaseTest, makes syntax corrections to BasePage

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates Jenkins xml

xXFranzFXx committed:
initial commit

sets up cucumber, gherkin
sets up test scenario

xXFranzFXx committed:
corrects cucumber-gherkin version to 7.9.0

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
updates LoginTests

Adds test cases to LoginTests (Smoke)

xXFranzFXx committed:
updates HomePage class

modifies HomePageTests, HomePage, LoginPage

xXFranzFXx committed:
updates LoginPage

syntax correction on build.gradle

xXFranzFXx committed:
Adds dataprovider

adds excel file for external data provider, changes AlbumsPage, LoginTests, BaseTest, adds Jenkins xml file

xXFranzFXx committed:
Update SmokeTests.xml

xXFranzFXx committed:
updates Homepage

xXFranzFXx committed:
Creates AllSongsPage

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
updates Homepage

xXFranzFXx committed:
updates HomPageTests

syntax correctin on HomePage

xXFranzFXx committed:
updates pages

xXFranzFXx committed:
adds ProfileTests

xXFranzFXx committed:
Adds AllSongsPage

adds AllSongsTests

xXFranzFXx committed:
Adds ArtistPage

creates ArtistsTests
syntax corrections

xXFranzFXx committed:
Adds AlbumsPage

creates AlbumsTests

xXFranzFXx committed:
updates HomeTests

syntax changes  build.gradle file

Berichicko commented:
Hello 
Homework 29 is cheeked!
Fantastic job! Magnificent! Terrific! 

💖I wish you successful and productive work!
💖It was truly a pleasure working and communicating with you!
💖To all of you keep on reaching for your goals, I believe in you!
💖Good luck and if you have questions, please reach out!
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/ef3991d6-3b52-4d03-8397-9cda60ad3787)

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/aba9a113-0930-4217-8c3d-3a8ecb147ad7)

https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/72a0012a-4bb9-4072-a2e8-95347b03d8cb

xXFranzFXx committed:
Syntax Corrections AlbumsPage

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Syntax corrections

corrects issue causing LoginTests to fail and POM not loading due to input required

xXFranzFXx committed:
syntax corrections
